src/ripple/core/Stoppable.h

```diff
@@ -186,13 +186,14 @@ class RootStoppable;
                                                              |
                                                          JobQueue
                                                              |
-        +--------+-----------+-----------+-----------+-------+---+----------+
-        |        |           |           |           |           |          |
-        |    NetworkOPs      |     InboundLedgers    |      OrderbookDB     |
-        |                    |                       |                      |
-     Overlay        InboundTransactions        LedgerMaster             Database
-        |                                            |                      |
-    PeerFinder                                LedgerCleaner            TaskQueue
+        +------+---------+--------+----------+---+----+---------+---+
```

Comment:
The smallest of tweaks: consider moving the third `+` from the end so its vertical line aligns with the `|` immediately below `JobQueue`.